indexing-service/src/main/java/org/apache/druid/indexing/common/task/Task.java

```diff
@@ -162,15 +162,21 @@ default int getPriority()
 
```

Comment:
Could you please add an overview section to `Task`'s class-level Javadoc, describing which methods of this interface could be called concurrently with which other methods and which could not? In other words, provide an overview [concurrent access documentation](https://github.com/code-review-checklists/java-concurrency#justify-document) for this interface.

Reply:
Raised https://github.com/apache/incubator-druid/issues/8271.